src/widgets/category/category-widget-model.js

```diff
@@ -136,10 +136,45 @@ module.exports = WidgetModel.extend({
     }
   },
 
+  sizeAcceptedCatsInData: function () {
```

Comment:
Since we're not really using any data from the widget model itself, they probably belong to CartoDB.js. Deep-insights.js could then use: `dataview.numberOfAcceptedCategories()` and `dataview.numberOfRejectedCategories()`. What do you think?